### PR TITLE
Blog&news css

### PR DESCRIPTION
**Pull request by JRybelis**
H3 text positioning and font size in responsive design.
Common text animation styling added to homepage.css; repeating code condensed.
CSS animation styling for news articles list.
Blog post picture thumbnail movement on hover.
fa calendar pseudo-element position adjusted to centre of the line. 
"Read more" link and fa arrow pseudo-element received uniform animation styling and be vertically aligned to the centre of the line.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -438,22 +438,22 @@ <h6 class=author-name>Steven King</h6>
         <div class="row">
             <div class="col-4 col-md-6 col-sm-8 col-xs-12">
```
Comment (pigma15) [inline index.html]: seems like ```div``` with ```<h6>``` inside might have col-md-12 col-sm-12 classes instead of col-md-6 col-sm-8

**Review by pigma15 — DISMISSED** [review]
Approveing since it does not break anything, however responsive design could be tweaked later

File: index.html (hunk 1 above)
Comment (bemze) [inline index.html]: 439 row - try to use col-md-8 instead 6 ant try responsive design. It worked for me.

**Review by bemze — COMMENTED** [review]
439 row - try to use col-md-8 instead 6 ant try responsive design. It worked for me.

**Review by martynas3113 — APPROVED** [review]
LGTM

**Review by bemze — APPROVED** [review]
LGTM

**Pull request merged**